.github/workflows/tests.yml

```diff
@@ -56,6 +56,33 @@ jobs:
           go generate -tags mysql ./...
           go test -v ./... -tags=mysql
 
+  go-release-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+      - uses: pnpm/action-setup@v2.2.2
+        with:
+          version: latest
+      - name: Cache MySQL
+        uses: actions/cache@v2
+        with:
+          path: |
+            ./resources/mysql/*.tar.gz
+            ./resources/mysql/*.tar.xz
+          key: ${{ runner.OS }}-build-mysql-cache
```

Comment:
Please keep consistent with `go-tests` steps.

Reply:
release mode needs to embed frontend https://github.com/bytebase/bytebase/blob/main/server/server_embed_release.go